2.6.0/spec/subscriptions.md

```diff
@@ -24,47 +92,56 @@ Response:
 HTTP/1.1 200 OK
 Content-Type: application/json
 ```
+
+#### Example for all subscriptions for all accesses
+
+When subscriptions for all accesses are requested the response contains a JSON-array with JSON-objects each containing the accessId and an array of subscriptions. 
+
+
 ```JSON
 [
   {
-    "subscriptionId": "8b2ad40b4ffd45e5b48425f57821a7eb",
     "accessId": "8732c2f065e2490babce820e94b1011a",
-    "service": "BB-1000-1000",
-    "operationalState": "ACTIVATED",
-    "spReference": "WloKMvmArcCFiV679uhWpAAtNgyvHxma",
-    "spSubscriptionId": "b49797aaed934642bc8ae136b87ed12b",
-    "option82": "52270123434F2D38373332633266303635653234393062616263653832306539346231303131610200",
-    "dhcpIdentifier": {
-      "ipv4CircuitId": "CO-8732c2f065e2490babce820e94b1011a"
-    },
-    "note": "Connection will be upgraded to 1000 Mbps at 2019-02-20",
-    "characteristics": {
-      "fixedIp": true,
-      "ipAddress": [
-        "1.2.3.4"
-      ],
-      "SLA": "SLA-3"
-    }
-  },
-  {
-    "subscriptionId": "af5143edb9624223810689c4100525f0",
-    "accessId": "8732c2f065e2490babce820e94b1011a",
-    "service": "IPTV",
-    "operationalState": "ACTIVATED",
-    "spReference": "WloKMvmArcCFiV679uhWpAAtNgyvHxma",
-    "spSubscriptionId": "55498978c4c446d0bbc2ae347a61e78d",
-    "option82": "52270123434F2D38373332633266303635653234393062616263653832306539346231303131610200",
-    "dhcpIdentifier": {
-      "ipv4CircuitId": "CO-8732c2f065e2490babce820e94b1011a"
-    },
-    "equipment": [
+    "subscriptions": [
```

Comment:
I dislike changing the format to array of object with key `subscriptions` holding an array of all subscriptions.
I would prefer we just keep the format as an array of subscriptions? Mainly this is to keep the subscription as the main subject of the endpoint but also to make it easier if I want to use it for for instance verifying invoices and the like.

Reply:
There is a separate InvoiceSpecification group that is working on the invoice parts and I hope that the resulting work they do take care of all verifications needs. As /subscriptions is a very short snapshot in time I would say the /subscriptions endpoint is not suitable for that purpose.

To me, we are iterating all accesses where we have one or more services so the main "array" should be accessId.
This is also to make all three endpoints iterate over the same concept. In /accesses the main array is centered on accessId and the same in /services. It would be strange if /subscriptions should deviate from that plan.